docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -1281,6 +1288,56 @@ Or also you could restrict your users to use only your DNS server where 192.168.
 
 image::docs/images/cisco-downloadable-acl-force-dns.png[scaledwidth="100%",alt="Force DNS ACL"]
 
+
+Web auth and Downloadable ACLs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It's possible to mix web authentication and downloadable ACLs starting from the IOS version 12.2, per example if you want to be able to redirect the device to the captive portal when a violation is triggered.
```

Comment:
starting from version 12.2 of the IOS. 

The second part with the per example is not really clear. Not sure what it means